modal.js

```diff
@@ -0,0 +1,244 @@
+    img: 'images/card1.png',
+    mobileImg: 'images/photo3.png',
+    technologies: ['html', 'css', 'ruby', 'javascript'],
+    Tags: ['Facebook', 'Full Stack Dev', '2015'],
+    tagsImg: ['./images/dot.png', './images/dot.png'],
+    liveLink: ['#'],
+    sourceLink: ['#'],
+  },
+  {
+    id: 4,
+    name: 'Uber Navigation',
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    moreDetails:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    img: 'images/card2.png',
+    mobileImg: 'images/photo4.png',
+    technologies: ['html', 'css', 'ruby', 'javascript'],
+    Tags: ['Uber', 'Lead Developer', '2015'],
+    tagsImg: ['images/dot.png', './images/dot.png'],
+    liveLink: ['#'],
+    sourceLink: ['#'],
+  },
+];
+const container = document.querySelector('#projects-section');
+const pop = document.querySelector('.try-me');
+
+function createCards() {
+  portfolioDetails.forEach((item) => {
+    container.innerHTML += `<div class="left1">
+  <div class="project-card">
+    <div class="snap1"><img src="${item.img}"></div>
+  </div>
+  <div class="left-block">
+    <h2 class="head">${item.name}</h2>
+    <div class="card-detail-box">
+      <ul>
+        <li class="card-detail canopy">${item.Tags[0]}</li>
+        <li class="card-detail option">
+          <img src="images/bullets.png" alt="bullet" class="circle" />
+          ${item.Tags[0]}
+        </li>
+        <li class="card-detail option">
+          <img src="images/bullets.png" alt="bullet" class="circle" />
+          2015
+        </li>
+      </ul>
+  
+      <p>
+      ${item.description}
+      </p>
+      <div class="language1-inline">
+        <ul>
+          <li class="langu">${item.technologies[0]}</li>
+          <li class="langu">${item.technologies[1]}</li>
+          <li class="langu">${item.technologies[2]}</li>
+        </ul>
```

Comment:
- [ ] Kindly, consider refactoring this part of your code in order to not repeat the same command 3 times; another reason is that if you have multiple technologies in the array, you want to display them all and not only your first 3 entries;

In order to refactor this, I recommend you to give a class to that `ul` list and create a function to target that new class (for ex with the document.querySelectorAll) and loop thru the array of objects and display all the technologies in that list of technologies.

Please bear in mind that at this moment we are not respecting the DRY principle: Do Not Repeat Yourself.
Please find here the [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), which you have listed also on the [Portfolio: details popup window](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) at the [HTML/CSS & JavaScript requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#htmlcss--javascript-requirements:~:text=HTML/CSS%20%26%20JavaScript%20requirements).